### PR TITLE
LPD-40825 Adjust selectable types

### PR DESCRIPTION
**Pull request by victorg1991**
There are already text that cover editable editing

### CODE REVIEW

**Comment by liferay-page-management** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperEmpty.js [hunk 1]
```diff
@@ -52,7 +53,7 @@ export default function ({activable = true, children, ...props}) {
 
 	if (Liferay.FeatureFlags['LPD-18221'] && activable) {
 		return (
-			<ActivableTopperEmptyWrapper {...props}>
+			<ActivableTopperEmptyWrapper ptyWrapper {...props}>
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperEmpty.js:56]: @victorg1991 ptyWrapper, good name, 😂

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperEmpty.js:56]: :O

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/TopperEmpty.js:56]: mega missclick

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

**Comment by victorg1991** [comment]
ci:test:page-management-playwright

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
@victorg1991 SF failure

**Comment by ealonso** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**